bedrock/careers/templates/careers/includes/testimonials.html

```diff
@@ -5,50 +5,52 @@
 #}
 {% block careers_content %}
   <section class="c-careers-testimonials c-careers-section no-js-hidden">
-    <div class="mzp-l-content">
-      <ul class="mzp-l-content mzp-l-columns mzp-t-columns-two c-careers-testimonials-wrapper">
-        <li class="has-modal" data-modal-id="perla-d">
-          <p>
-            "Despite a diverse set of backgrounds, we share a similar passion and need for data and browser transparency..."
-          </p>
+      <ul class="mzp-l-content c-careers-testimonials-wrapper">
+        <li class="c-careers-testimonial-item has-modal" data-modal-id="perla-d">
+          <q>
+            Despite a diverse set of backgrounds, we share a similar passion...
```

Comment:
**Suggestion:** if we reduce the font size a bit, we'd be able to squeeze in the full pull-quotes into the squares without having to set a min-height. Let me comment below respectively.

Reply:
Nit: We could use the unicode ellipsis character instead of three periods: … 
I'm not sure what how much real difference it makes but to me it just looks "neater"